pkg/controllers/state/cluster.go

```diff
@@ -522,7 +526,7 @@ func (c *Cluster) updateNodeUsageFromPod(ctx context.Context, pod *v1.Pod) error
 	n, ok := c.nodes[c.nameToProviderID[pod.Spec.NodeName]]
 	if !ok {
 		// the node must exist for us to update the resource requests on the node
-		return errors.NewNotFound(schema.GroupResource{Resource: "Machine"}, pod.Spec.NodeName)
+		return errors.NewNotFound(schema.GroupResource{Resource: "Node"}, pod.Spec.NodeName)
```

Comment:
Shouldn't this be NodeClaim since that's what we're renaming it to be?

Reply:
No, this should have been `Node` originally but this was incorrectly put as `Machine` when this was originally merged.